src/library/BatchKeys/index.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 @paritytech/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+export const batchkeys: Array<string> = [];
```

Comment:
Change to `batchKeys`. We like the camel case convention. This rule is commented out in the linter but will be enabled as we work through more refactoring.